content/en/docs/reference/using-api/server-side-apply.md

```diff
@@ -561,14 +561,6 @@ Downgrading works because kubectl Server-Side Apply keeps the
 `last-applied-configuration` annotation up-to-date if you use
 `kubectl apply`.
 
-This behavior applies to Server-Side Apply with the `kubectl` field manager.
-As an exception, you can opt-out of this behavior by specifying a different,
-non-default field manager, as seen in the following example. The default field
-manager for kubectl server-side apply is `kubectl`.
-
-```shell
-kubectl apply --server-side --field-manager=my-manager [--dry-run=server]
-```
```

Comment:
Is this repeating text intentional?

Reply:
I guess it's not. It's talking about upgrade from client-side-apply to server-side-apply, it belongs to pre sections.

Reply:
@tengqm Any suggestions? I not very sure about this.

Reply:
I'm not that familiar with SSA either. Sorry.

Reply:
The original content is not entirely clear. Best to ask for a technical review.

Reply:
Sure, but how can I request for a technical review? @kbhawkey